### DISCUSSION TITLE
Client option to prevent saving password

### DISCUSSION THREAD

**Discussion by KrumJ12** [rustdesk/rustdesk#8158] [General]
Hi,
we are using Rustdesk on premise server and we would like an option, that client cannot save the password when connecting to another client with permanent access and password set. This is our wish due to security concerns.

Is it possible?

**Comment by angeloxo** [comment]
Agree with you @KrumJ12 . 
Anydesk has that opcion aswell.
I use to check this option with Anydesk, very interesting for avoid "cache" passwords on remote computers.
